airflow/providers/cncf/kubernetes/utils/pod_manager.py

```diff
@@ -480,7 +480,8 @@ def consume_logs(*, since_time: DateTime | None = None) -> tuple[DateTime | None
                             self._callbacks.progress_callback(
                                 line=line, client=self._client, mode=ExecutionMode.SYNC
                             )
-                    self.log.info("[%s] %s", container_name, message_to_log)
+                    if message_to_log and message_to_log.strip():
+                        self.log.info("[%s] %s", container_name, message_to_log)
```

Comment:
What if the container logs empty lines on purpose, unusual in logs but something like a print("\n") for spacing?
We would expect to still get this line?

I assumed we would only need to test if `message_to_log is not None`, as `message_to_log = None` explicitly tells us no line got fetched from the container, whereas a `message_to_log == ""` could still mean a log line got fetched from the container (as expected), even if the actual line is an empty string.

This would work for my case but I wonder if this could create a side effect for users working with applications that space their output with empty line.

Reply:
hmm, ok so it does str strip here https://github.com/apache/airflow/blob/d3ef6736eafa96b68032099d4158911e630e9895/airflow/providers/cncf/kubernetes/utils/pod_manager.py#L628 so strip here again is redundant and may not require. 
Actually, I also find this None log a bit weird and to me it look like a bug